### PR TITLE
ACM#11233: Part 2: Simplify user-replaced values and commands

### PR DESCRIPTION
**Pull request by xenolinux**
https://issues.redhat.com/browse/ACM-11233

### CODE REVIEW

File: clusters/hosted_control_planes/disable_hosted.adoc [hunk 1]
```diff
@@ -6,46 +6,49 @@ You can uninstall the HyperShift Operator and disable the hosted control plane.
...
-. Disable the `hypershift-addon` by running the following command:
+. Disable the `hypershift-addon` managed cluster add-on by running the following command:
 +
 ----
-oc patch mce multiclusterengine --type=merge -p '{"spec":{"overrides":{"components":[{"name":"hypershift-local-hosting","enabled": false}]}}}' <1>
+oc patch mce <multiclusterengine-instance-name> --type=merge -p '{"spec":{"overrides":{"components":[{"name":"hypershift-local-hosting","enabled": false}]}}}' <1>
 ----
 +
-<1> The default `MultiClusterEngine` resource instance name is `multiclusterengine`, but you can get the `MultiClusterEngine` name from your cluster by running the following command: `$ oc get mce`.
+<1> The default `MultiClusterEngine` resource instance name is `multiclusterengine`. You can get the `MultiClusterEngine` instance name from your hosting cluster by running the `$ oc get mce` command.
```
Comment (jparrill) [inline clusters/hosted_control_planes/disable_hosted.adoc]: Could you please change `hosting` by `hosted`?

File: clusters/hosted_control_planes/disable_hosted.adoc [hunk 2]
```diff
@@ -6,46 +6,49 @@ You can uninstall the HyperShift Operator and disable the hosted control plane.
...
+
+. Run the following command to disable the hosted control planes feature:
 
 ----
-oc patch mce multiclusterengine --type=merge -p '{"spec":{"overrides":{"components":[{"name":"hypershift","enabled": false}]}}}' <1>
+oc patch mce <multiclusterengine-instance-name> --type=merge -p '{"spec":{"overrides":{"components":[{"name":"hypershift","enabled": false}]}}}' <1>
 ----
 
-<1> The default `MultiClusterEngine` resource instance name is `multiclusterengine`, but you can get the `MultiClusterEngine` name from your cluster by running the following command: `$ oc get mce`.
+<1> The default `MultiClusterEngine` resource instance name is `multiclusterengine`. You can get the `MultiClusterEngine` name from your hosting cluster by running the `$ oc get mce` command.
```
Comment (jparrill) [inline clusters/hosted_control_planes/disable_hosted.adoc]: Could you please change `hosting` by `hosted`?

File: clusters/hosted_control_planes/disconnected_intro.adoc [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
 
 image:../images/489_RHACM_HyperShift_on_bare_metal_1223.png[Disconnected architecture diagram]
 
-. You configure infrastructure services, including the registry certificate deployment with TLS support, web server, and DNS, to ensure that the disconnected deployment works.
-. You create a config map in the `openshift-config` namespace. In this example, the config map is named `registry-config`. The content of the config map is the Registry CA certificate. The data field of the config map must contain the following key/value: 
+. Configure infrastructure services, including the registry certificate deployment with TLS support, web server, and DNS, to ensure that the disconnected deployment works.
+. Create a config map in the `openshift-config` namespace. The content of the config map is the Registry CA certificate. The data field of the config map must contain the following key and value:
 
-* Key: `<registry_dns_domain_name>..<port>`
+* Key: `<registry-dns-domain-name>..<port>`
```
Comment (jparrill) [inline clusters/hosted_control_planes/disconnected_intro.adoc]: Could you please remark emphasis on the ".." (the two dots after the registry name) in order to let the customer know that is replacing the ":"?

**Review by jparrill — CHANGES_REQUESTED** [review]
Dropped some comments

**Review by jparrill — APPROVED** [review]
/lgtm

**Review by swopebe — APPROVED** [review]
This looks great, obviously it was much needed.

**Pull request merged**